docs/agent/changelog.md

```diff
@@ -6,6 +6,12 @@ title: Changelog
 
 ## v3
 
+### ngrok Agent 3.14.0 - \[2024-08-01\]
+
+- Added `--traffic-policy-file` flag that accepts Traffic Policy configuration for HTTP, TCP, or TLS traffic.
```

Comment:
are these also updated in https://ngrok.com/docs/agent/cli/ and https://ngrok.com/docs/agent/config/ ?

Reply:
no but they will be. since the old fields still work for the time being, there is no absolute hurry. there is possibly an incentive to hold off, as these new names aren't present for old agent version. writing these doc changes was my next goal. However, if people think we should just do it as part of this PR, I can add it in.

Reply:
resolving because these doc changes either already existed or were added to the PR